### PR TITLE
Enrich logging, minor refactoring

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: static/js/app.js [hunk 1]
```diff
@@ -1,57 +1,52 @@
...
+  //if pressed with alt key...
   if (event.altKey) {
+    //...then remove the annotation
     let position = textArea[0].selectionStart;
     tagModel.removeAnnotation(position);
-  } else {
-    if (tagModel.currentCategory === null) {
-      alert('Error: Please select a label and highlighter color first');
-      return;
-    }
```
Comment (tmarkn) [inline static/js/app.js]: we're going to need this later when we can delete labels

File: static/js/app.js [hunk 2]
```diff
@@ -1,57 +1,52 @@
...
     let range = {
       'startPosition': textArea[0].selectionStart,
       'endPosition': textArea[0].selectionEnd
     };
+    if (range.startPosition !== range.endPosition) {
+      console.log("Highlighted: " + range.startPosition + "-" + range.endPosition);
+    }
     if (range.startPosition < range.endPosition) {
-      tagModel.addAnnotation(range, tagModel.currentCategory);
+      tagModel.addAnnotation(range);
```
Comment (tmarkn) [inline static/js/app.js]: we can merge these

**Review by tmarkn — APPROVED** [review]
good improvements throughout

**Review by keithudock — APPROVED** [review]
Console logs a label has been selected twice when selecting the color dropper. Defect reported. Doesn't affect functionality for demo

**Pull request merged**